docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -199,7 +198,7 @@
S0RkkohMWrfGvlRGREREtDeJlIWnqkUAPgBwrKoWaqAYwJMABjvbPKqqA1V1YHZu3m4PmIiIiKiuVScLL19EssLv0wAcDeA7EWkexgTAKQD8mhxERERE9Uh1svCaAxgtIokIJlwvquqbIjJBRPIBCICpAC6rvWESERER7Tmqk4X3DYB+RvyIqJ1tLy3D9+sqLw0wbblfaHfphm1mfFuJnQg4vJdfUPaJyYvN+KFt7OUCUhL9tPgV6+1U9jbOkgSdmmW4be0osZdX+HjJWneb1rl2P2/NXm7GT+nuH5fmztIHpU7V3s+dZR8A4Njuzc34iBPtArhfLLKLQgPAsEPam/Gezex9n7HcLooMAJudAsirZs+x4/P8lPxR/3uSPa58+zX+bKpfFHpbib3sQ9tWjd1tvPeYVxjYW6oAANZ+8b4ZLy07yox7hZwB4OGnP/cfJCKqZ7gSOREREVFEnEARERERRcQJFBEREVFEnEARERERRcQJFBEREVFEEpS6i4+23XrrdU+8Xil+bOdm7jZeQdvpi9eb8bHfrXDbuvHo/cz4nMKNZrxpYzs7DQDemWtnuzVKtgsjJyf4c9WWmXZh5JIYKU8tsv2xWWYt97PdtuywM8EaJNgViw9on+u2dd/Hdvba8B52dt6xt49323r6qkPNeHKifSybNfKPSZ+RD5vxb56+3IznZfqViTZstbNGP1q4yoyf2qul29YHc+xtmqb7+1JSamdt/nrMVDP+9IWD3La8TMv9T77O3iDTf+0nPvs7M96nTaPJqjrQ3ZCIaC/EM1BEREREEXECRURERBQRJ1BEREREEXECRURERBQRJ1BEREREEVWnmHCNSUlMQKesyvXCMlP9YZQ5WYKNG9rZbu1y/OwpT1pyohlPaeDPL5um2VlSr85aacbP7u3Xokt0st1Skvz+3W2cMSfFyAIsKrZr/n1WYGcnHtop321r63Y7Qyw7PdmMd+nmZ2A2zbSPcVKive9bt9vZhACAZLt+nDcu7z0BAF7maqesTL9/R1oDu59Gaf7PxMyVdkZl2xaNzHhuhr2PQIzadl623cY1bluN0uyfSSKi+ohnoIiIiIgi4gSKiIiIKCJOoIiIiIgi4gSKiIiIKCJOoIiIiIgi4gSKiIiIKKK4FhPu2qufPv7KhErxl2b5BYDnFtop20d3t1Pph/f2C7fe+M4cM37pwNZmvKh4u9tWwwZ2mnmnppWXaQBiL9WQ4CxJcOHzU91t/nhEJzP+l//axXxvPcYupAwAGc7YikvsJQn++J/Zblt3n9TNjC9cudmMvzzLLsoMAGf2tJd+KNy4zYw3TPKXHvAKDf/s5nfM+Po1drFqAHj+xuPMeF6avYTGCzML3baO7JBjxneU2cceAIZ2bWrGfzlmmhlvlWMXqwaAh5/+3Ix/ft8ZZjzWUgXdjraLCW+b+hCLCRNRvcMzUEREREQRcQJFREREFBEnUEREREQRcQJFREREFBEnUEREREQRxTULT0RWAfg+/G8egNVx67yyfbn/fXnf67r/fXHf26qqX4GaiGgvFNcJ1E86FplUl6nN+3L/+/K+13X/+/K+ExHVJ7yER0RERBQRJ1BEREREEdXlBOrROux7X+9/X973uu5/X953IqJ6o87ugSIiIiLaW/ESHhEREVFEdTKBEpFjRWS2iMwTkWvj3HeBiEwXkakiMikO/T0hIitFZEaFWI6IjBeRueG/2XHuf5SILA2PwVQROb6W+m4tIh+IyLciMlNErgrjcdn/GP3Ha/9TReRLEZkW9n9zGG8vIhPD9/8LIpIcx76fEpGFFfa9b033TUS0L4j7JTwRSQQwB8DRAJYA+ArACFX9Nk79FwAYqKpxWQtHRA4FsAnA06raM4z9BcBaVb0znEBmq+of4tj/KACbVPWe2uizQt/NATRX1a9FJBPAZACnADgfcdj/GP0PR3z2XwCkq+omEUkC8AmAqwD8FsArqjpGRB4BME1VH45T35cBeFNVX6rJ/oiI9jV1cQZqMIB5qrpAVbcDGANgWB2MIy5U9SMAa3cKDwMwOvx+NIIP9Xj2HxeqWqiqX4ffbwQwC0BLxGn/Y/QfFxrYFP43KfxSAEcAKJ/A1Mr+x+ibiIhqQF1MoFoCWFzh/0sQxw81BB8i74rIZBG5JI79VtRUVQvD75cDaFoHY7hSRL4JL/HV2iXEciLSDkA/ABNRB/u/U/9AnPZfRBJFZCqAlQDGA5gPoEhVS8Kn1Nr7f+e+VbV8328P9/1+EUmpjb6JiOq7ffEm8oNVtT+A4wBcEV7iqjMaXEON95mBhwF0BNAXQCGAe2uzMxHJAPAygKtVdUPFx+Kx/0b/cdt/VS1V1b4AWiE4+9q1tvqqqm8R6QngunAMgwDkAKiVS8dERPVdXUyglgJoXeH/rcJYXKjq0vDflQDGIvhQi7cV4f055ffprIxn56q6IvxwLQPwGGrxGIT337wM4FlVfSUMx23/rf7juf/lVLUIwAcADgCQJSINwodq/f1foe9jw8uaqqrFAJ5E3bz/iYj2enUxgfoKQOcwEykZwJkAXo9HxyKSHt5MDBFJBzAUwIzYW9WK1wGMDL8fCeC1eHZePnkJnYpaOgbhjcyPA5ilqvdVeCgu++/1H8f9zxeRrPD7NASJE7MQTGbOCJ9WK/vv9P1dhYmrILj3qi7e/0REe706WUgzTBt/AEAigCdU9fY49dsBwVknAGgA4Lna7ltEngdwOIA8ACsA3ATgVQAvAmgD4HsAw1W1Vm70dvo/HMHlKwVQAODSCvck1WTfBwP4GMB0AGVh+HoE9yHV+v7H6H8E4rP/vRHcJJ6I4I+VF1X1lvB9OAbBJbQpAM4JzwjFo+8JAPIBCICpAC6rcLM5ERFVE1ciJyIiIopoX7yJnIiIiGi3cAJFREREFBEnUEREREQRcQJFREREFBEnUEREREQRcQJFREREFBEnUEREREQRcQJFREREFNH/AzpI7bdG1vdCAAAAAElFTkSuQmCC\n",
```

Comment:
This text below here was like this before

>  "`qiskit` also contains the `qsvc` class that extends the `sklearn svc` class, that can be used as follows:"

Arguably the class is `QSVC` in capitals. and the sklearn one `SVC`.

Reply:
Replaced with "Qiskit Machine Learning also contains the `QSVC` class that extends the `SVC` class from scikit-learn,"